trunk/src/app/srs_app_rtc_dtls.cpp

```diff
@@ -338,7 +338,7 @@ srs_error_t SrsDtlsCertificate::initialize()
         /*int r = */X509_digest(dtls_cert, EVP_sha256(), md, &n);
```

Comment:
The value of n may exceed the capacity of the floating-point number, making it larger than its intended size.